.github/workflows/helm_chart.yaml

```diff
@@ -0,0 +1,140 @@
+  #
+  # Note: currently ${{ env }} cannot define dynamically initialized values.
+  environment:
+    name: Configure environment
+    runs-on: ubuntu-latest # TODO: until static global env can be used through ${{ env }}.
+    outputs:
+      git_refname: ${{ steps.set-git-refname.outputs.git_refname }}
+      github_runner_default_os: ubuntu-latest
+      helm_push_enabled: ${{ steps.set-helm-push-enabled.outputs.helm_push_enabled }}
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Set Git refname
+        id: set-git-refname
+        run: |
+          GIT_REFNAME="$(echo "${{ github.ref }}" | sed -r 's@refs/(heads|pull|tags)/@@g')"
+
+          echo "GIT_REFNAME=${GIT_REFNAME}"
+          echo ::set-output name=git_refname::${GIT_REFNAME}
+
+      - name: Set Helm push enabled
+        id: set-helm-push-enabled
+        run: |
+          HELM_PUSH_ENABLED=""
+          if [ "${{ github.event_name }}" == "push" ] && echo "${{ steps.set-git-refname.outputs.git_refname }}" | grep -E -q "^chart/${{ env.HELM_CHART_NAME }}/[0-9]+.[0-9]+.[0-9]+**"; then
+            HELM_PUSH_ENABLED=1
+          else
+            printf >&2 "Unstable chart (%s) from %s event, chart will not be pushed" "${{ steps.set-git-refname.outputs.git_refname }}" "${{ github.event_name }}"
+          fi
+
+          echo "HELM_PUSH_ENABLED=${HELM_PUSH_ENABLED}"
+          echo ::set-output name=helm_push_enabled::${HELM_PUSH_ENABLED}
+
+  helm:
+    name: Helm
+    needs:
+      - environment
+    runs-on: ${{ needs.environment.outputs.github_runner_default_os }}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          # Required for correctly populating ldflags
+          fetch-depth: 0
+
+      - name: Setup Helm
+        run: |
+          ARCH=$(uname -m) || exit 1
+          case "${ARCH}" in
+            aarch64_be | aarch64 | armv6l | armv7l | armv8b | armv8l)   ARCH=arm64 ;;
+            x86_64)                                                     ARCH=amd64 ;;
+            *)                                                          printf >&2 "unsupported architecture: %s\n" "${ARCH}" ; exit 1 ;;
+          esac
+
+          KERNEL_NAME=$(uname -s | tr "[:upper:]" "[:lower:]") || exit 1
+          HELM_DIRECTORY_NAME="helm-v${{ env.HELM_VERSION }}-${KERNEL_NAME}-${ARCH}"
+          HELM_DIRECTORY_PATH="${{ github.workspace }}/${HELM_DIRECTORY_NAME}"
+          HELM_ARCHIVE_NAME="${HELM_DIRECTORY_NAME}.tar.gz"
+          HELM_ARCHIVE_PATH="${{ github.workspace }}/${HELM_ARCHIVE_NAME}"
+
+          curl -sSL "https://get.helm.sh/${HELM_ARCHIVE_NAME}" -o "${HELM_ARCHIVE_PATH}"
+          mkdir -p "${HELM_DIRECTORY_PATH}"
+          tar -xzf "${HELM_ARCHIVE_PATH}" -C "${HELM_DIRECTORY_PATH}"
+
+          echo "${HELM_DIRECTORY_PATH}/${KERNEL_NAME}-${ARCH}" >> $GITHUB_PATH
+
+      - name: Add Helm repositories
```

Comment:
How about just running the three `helm repo add` commands? To read what this does I have to go up looking for the parameters and I see little value in parametrizing this step.

Reply:
Fixed in 9f64111.